antora.yml

```diff
@@ -8,20 +8,23 @@ asciidoc:
   attributes:
     product_name: OpenShift Serverless Logic
     context: OpenShift Serverless Logic
-    kogito_version_redhat: 1.24.0.Final-redhat-00011
+    kogito_version_redhat: 1.27.0.Final-redhat-00001
     quarkus_platform: com.redhat.quarkus.platform
     kogito_sw_ga: >-
       org.kie.kogito:kogito-quarkus-serverless-workflow
     quarkus_version: 2.7.6.Final-redhat-00006
     java_min_version: 11+
     maven_min_version: 3.8.1
-    graalvm_min_version: 22.1.0
+    graalvm_min_version: 22.2.0
```

Comment:
DP2 is still using Quarkus 2.7 which uses GraalVM 21.3

Reply:
Done!